.github/workflows/python-dev.yml

```diff
@@ -23,12 +23,14 @@ name: Python Dev
 on:
   push:
     branches:
-      - main
-      - 1.5.x
+#      - main
+#      - 1.5.x
+      - None
   pull_request:
     branches:
-      - main
-      - 1.5.x
+#      - main
+#      - 1.5.x
+      - None
```

Comment:
@phofl I'm confused, why comment these out?

Reply:
My reasoning was that it would be easier to add them back in for 3.12, but no strong opinion, can remove if you prefer

Reply:
I think previously we would comment/uncomment `if: false` below to temporarily disable this workflow

Reply:
ah ok - that's fine